Makefile

```diff
@@ -37,6 +37,13 @@ _bootstrap-python:
 _bootstrap-githooks: clean-githooks
 	-(cd ${ROOTDIR}; scripts/dev/git-hooks.sh --install)
 
+# This should only be needed for CI rather than locally,
+# so we don't run it in `make bs`
+.PHONY: bootstrap_aws_creds
+bootstrap_aws_creds:
+	mkdir -p ~/.aws
+	echo ${AWSBase64Creds} | base64 -d > ~/.aws/credentials
```

Comment:
where does `AWSBase64Creds` come from?

Reply:
right now we just insert it as an environment variable in circleCI for each repo but i think we're moving towards using circleCI contexts https://circleci.com/docs/2.0/contexts/